cloudpak/.travis.yml

```diff
@@ -0,0 +1,47 @@
+---
+dist: xenial
+sudo: required
+group: bluezone
+language: generic
+
+# Environment variables (setup endpoint to IBM github)
+env:
+  global:
+  - OCTOKIT_API_ENDPOINT="https://github.ibm.com/api/v3/"
+
+before_install:
+  # Clone build tools & set tools path for follow on processing
+  # The clone command will use the ssh key from the travis settings to clone the repo from github.ibm.com
+  - if [[ ! -z "$BRANCH_OVERRIDE" ]] ; then
+      git clone -b $BRANCH_OVERRIDE git@github.ibm.com:ibmprivatecloud/content-tools;
+    else 
+      git clone git@github.ibm.com:ibmprivatecloud/content-tools;
+    fi
+  - export toolsPath=`pwd`/content-tools/travis-tools/
+  # Install dependencies & determine chart delta 
+  - $toolsPath/build/bin/installDependencies.sh
+  - export changeList=`$toolsPath/build/bin/determineChartDelta.sh | tee determineChartDelta.out | grep 'return determineChartDelta:' | cut -f2 -d:` && cat determineChartDelta.out
+   
+install:
+  # Package for release 
+  - if [[ ! -z "$TRAVIS_TAG" ]] ; then $toolsPath/release/bin/package.sh; fi
+  # Lint and install/test charts (if cv-tests exist)
+  - $toolsPath/cv-test/bin/validateContent.sh
+
+script: echo "Override default script"
+
+deploy:
+  # scp helm repo(s) to location identified (Note: SSHPASS env variable must contain password)
+  - provider: script
+    skip_cleanup: true
+    script:  $toolsPath/build/bin/deployHelmRepo.sh
+    on:
+      all_branches: true
+  # Publish tagged release
+  - provider: releases
+    skip_cleanup: true
+    api_key: $GITHUB_TOKEN
+    file_glob: true
+    file: repo/stable/*
+    on:
+      tags: true
```

Comment:
I am not sure that this file will end up needing to be checked in here, but I don't think it should cause an issue if it does go in with the rest of the changes.

Reply:
Sure, If you want, I can remove that. Please let me know.